.changelog/c898d340-9c15-4b9a-a5da-93a340a9d5f8.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "c898d340-9c15-4b9a-a5da-93a340a9d5f8",
+  "type": "bugfix",
+  "collapse": true,
+  "description": "transport/http: fix metrics race condition",
+  "modules": [
+    "transport/http"
```

_modules_, not packages. The root smithy-go module is "."

```suggestion
    "."
```